src/main/java/io/electrum/vas/model/QrPayment.java

```diff
@@ -0,0 +1,102 @@
+package io.electrum.vas.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.electrum.vas.Utils;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+import javax.validation.constraints.NotNull;
+import java.util.Objects;
+
+/**
+ * @since v3.31.0
+ */
+@ApiModel(description = "Model for QR-based payments. "
+      + "This payment method should be used when a QR code is presented for payment.", parent = PaymentMethod.class)
+public class QrPayment extends PaymentMethod {
+
+   private String tranId = null;
+
+   private String partnerPaymentToken = null;
+
+   public QrPayment() {
+      setType(PaymentMethodType.QR);
+   }
+
+   @ApiModelProperty(required = true, value = "The unique transaction identifier related to this transaction. The tranId is embedded in sub-Tag 00 of the Electrum MAIT.")
```

Comment:
MAITs only mean something in the context of EMVCo.

```suggestion
   @ApiModelProperty(required = true, value = "The unique transaction identifier related to this transaction. In QRs styled on the EMVCo specification, the tranId is embedded in sub-Tag 00 of the Electrum MAIT.")
```